apps/airtable-cache/src/app.ts

```diff
@@ -0,0 +1,226 @@
+import { fastify, FastifyReply, FastifyRequest } from 'fastify';
+import { fastifyCors } from '@fastify/cors';
+import axios, { AxiosResponse } from 'axios';
+import { errorHandlerPlugin } from './lib/errorHandlerPlugin';
+import { sha256 } from './lib/sha256';
+import {
+  CacheValue, clearValues, clearValuesContainingBody, getValue, setValue,
+} from './lib/cache';
+
+export const getInstance = async () => {
+  const instance = fastify();
+
+  await instance.register(errorHandlerPlugin);
+  await instance.register(fastifyCors);
+
+  await instance.register(async (i) => {
```

Comment:
At the moment, this is a free proxy for any requests to Airtable, and will cache anybody's requests there.

We should instead validate the token is from some allowlisted set.